Button.css

```diff
@@ -0,0 +1,36 @@
+.btn {
+    margin : 1.25rem;
+    font-family: "comic sans ms";
+    font-weight: 50%;
+    width : 12.5rem;
+    height : 3.125rem;
+    font-size: 1.125rem;
+    background-color: aqua;
+    
+    border : 0.125rem solid white;
+    border-radius: 1.25rem;
+
+}
+
+.txt-style{
```

Comment:
You can move this txt-style to your utility class, as it will be used in multiple places.